### PR TITLE
list trieste as downstream project

### PR DESCRIPTION
**Pull request by joelberkeley**
**PR type:** doc improvement

## Summary

**Proposed changes**
List the trieste project

### Release notes

**Fully backwards compatible:** yes

**Commit message (for release notes):**

list trieste as downstream project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ A few projects building on GPflow and demonstrating its usage are listed below.
 | Project | Description |
 | --- | --- |
 | [GPflowOpt](https://github.com/GPflow/GPflowOpt)       | Bayesian Optimization using GPflow. |
+| [Trieste](https://github.com/secondmind-labs/trieste)       | Bayesian optimization with TensorFlow, with out-of-the-box support for GPflow models. |
```
Comment (joelberkeley) [inline README.md:110]: i put it with gpflowopt as it's a similar topic. Does that seem right?

Comment (st--) [reply README.md:110]: I'm happy for Trieste to be at the top together with GPflowOpt!

File: README.md [hunk 2]
```diff
@@ -107,6 +107,7 @@ A few projects building on GPflow and demonstrating its usage are listed below.
 | Project | Description |
 | --- | --- |
 | [GPflowOpt](https://github.com/GPflow/GPflowOpt)       | Bayesian Optimization using GPflow. |
```
Comment (st--) [inline README.md:109]:
```suggestion
| [GPflowOpt](https://github.com/GPflow/GPflowOpt)       | Bayesian Optimization using GPflow (does not support gpflow >= 2.0 in master). |
```
?

File: README.md (hunk 1 above)
Comment (st--) [inline README.md:110]:
```suggestion
| [Trieste](https://github.com/secondmind-labs/trieste)       | Bayesian optimization with TensorFlow, with out-of-the-box support for GPflow (2.x) models. |
```
(together with other suggestion above) to clarify which one you might want to use when ?

File: README.md (hunk 2 above)
Comment (joelberkeley) [reply README.md:109]: sure, though it doesn't support gpflow 1.x either

Comment (st--) [reply README.md:109]: changed it

**Review by st-- — APPROVED** [review]

**Pull request merged**